deps/wxWidgets/wxWidgets.cmake

```diff
@@ -28,9 +28,8 @@ endif ()
 
 
 bambustudio_add_cmake_project(wxWidgets
-    GIT_REPOSITORY "https://github.com/wxWidgets/wxWidgets"
+    GIT_REPOSITORY "https://github.com/scottmudge/wxWidgets-bambustudio"
```

Comment:
I agree the patch is not an elegant solution.
Though I think we have issues here: different OS requires different patches :(
From the maintenance point of view, the patch is still easier.

Reply:
Ah, yeah there are some OS switches in the CMake script. Forgot about that. Alright, I'll keep this change local for now.